### ISSUE TITLE
Plot SAILS scans

### ISSUE THREAD

**Issue by freemansw1** [freemansw1/radar-plotter#1]
Right now, the script will only plot one sweep at a time from the volume files. Unfortunately, NEXRAD can sometimes include multiple sweeps of interest in the same volume file (e.g., for SAILS). We should loop through the available sweeps and plot multiple times if they are available.

**Comment by ldgrantcsu** [comment]
It would be great if this could be kept flexible though. E.g. for plotting radar throughout the entirety of BACS, I already have 6500 files just with reflectivity from KCYS and I actually don't want or need any higher time resolution than this!

**Comment by freemansw1** [comment]
Yes, totally agreed. That would give the flexibility to plot high-resolution temporal data for specific cases while keeping the lower-resolution data for the broader overview.
